src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -3794,6 +3794,9 @@ You should consider suppressing the warning only if you're sure that you don't w
   <data name="ERR_SignButNoPrivateKey" xml:space="preserve">
     <value>Key file '{0}' is missing the private key needed for signing</value>
   </data>
+  <data name="ERR_PublicSignNoKey" xml:space="preserve">
```

Comment:
C# actually already had this message and a test that verified it was produced (`PublicSignNoKey` in `InternalsVisibileToAndStrongNameTests.cs`) -- this was just an extraneous addition to C# and a missing message in VB. I've changed it to use the suggested helper and it fails appropriately unless I add the correct message to the VB resources.